CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Change Log
 
+### Added
+## 6.0.0 (Unreleased)
+-   `unitSpace` prop to `<pxb-channel-value>` to manage spacing between the value and units.
```

Comment:
In angular, it is called a directive instead of a prop, I believe.

Reply:
Following old format https://github.com/pxblue/angular-component-library/blob/dev/CHANGELOG.md#added-1